doc/languages-frameworks/go.section.md

```diff
@@ -26,6 +27,7 @@ pet = buildGoModule rec {
     sha256 = "0m2fzpqxk7hrbxsgqplkg7h2p7gv6s1miymv3gvw0cz039skag0s";
   };
 
+  modRoot = ".";
```

Comment:
The default is . so we do not need to specify it.

Reply:
This specific doc example it is "." but it is common to need to override with Go modules so it should be in the documentation.

Reply:
Maybe we add a special note so people don't have to go into the Nic lambda (which they often won't) to look up how to set this up for typical Go modules (not the most basic pet example here)? This is a very basic example that doesn't represent a lot of Go modules in the wild.

Reply:
> This specific doc example it is "." but it is common to need to override with Go modules so it should be in the documentation.

Maybe we find a better example where it isn't the default value?

> Maybe we add a special note so people don't have to go into the Nix lambda

What do you mean exactly?